publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java

```diff
@@ -659,7 +659,7 @@ protected List<AccessRight> getCuratorAccessRightsWithDegree() {
         return accessRights;
     }
 
-    protected List<AccessRight> getCuratorAccessRightsWithEmbargoDegree() {
+    protected List<AccessRight> getAccessRightsForEmbargoThesisCurator() {
```

Comment:
Eeeeegentlig burde de over også skifte navn. Men det får være opp til deg :)

Reply:
your wish is my command! :-)